plugins/inputs/opcua_listener/README.md

```diff
@@ -147,13 +188,17 @@ to use them.
   ##   example: default_tags = { tag1 = "value1" }
   # default_tags = {}
   #
+  ## Group default sampling interval. If a node in the group doesn't set its
+  ## sampling interval, this is used.
+  # sampling_interval = 
```

Comment:
Should this include a default value or example value rather than an empty =?

Reply:
Ok, I would suggest setting it to 0 as well.